README.md

```diff
@@ -18,7 +18,8 @@ src
 ├─ voting-strategies
 │  ├─ CompVotingStrategy.sol - "Strategy that uses delegated balances of Comp tokens as voting power"
 │  ├─ OZVotesVotingStrategy.sol - "Strategy that uses delegated balances of OZ Votes tokens as voting power"
-│  ├─ WhitelistVotingStrategy.sol — "Strategy that gives predetermined voting power for members in a whitelist, otherwise zero"
+│  ├─ WhitelistVotingStrategy.sol — "Strategy that gives predetermined voting power for members in a whitelist, otherwise zero. Whitelist is stored in a bytes array On-Chain."
+│  ├─ MerkleWhitelistVotingStrategy.sol — "Strategy that gives predetermined voting power for members in a whitelist, otherwise zero. Whitelist is stored in a Merkle tree Off-Chain, with only the root being stored On-Chain."
```

Comment:
I don't think we should write `otherwise zero`, as the contract will [revert](https://github.com/snapshot-labs/sx-evm/blob/4a693e4adee064c2dea42b44719bb4a860fe3fe6/src/voting-strategies/MerkleWhitelistVotingStrategy.sol#L42) in that case

Reply:
Agreed